Include/internal/pycore_code.h

```diff
@@ -283,7 +283,7 @@ extern int _PyStaticCode_Init(PyCodeObject *co);
     do { if (_Py_stats && PyFunction_Check(callable)) _Py_stats->call_stats.eval_calls[name]++; } while (0)
 #define GC_STAT_ADD(gen, name, n) do { if (_Py_stats) _Py_stats->gc_stats[(gen)].name += (n); } while (0)
 #define OPT_STAT_INC(name) do { if (_Py_stats) _Py_stats->optimization_stats.name++; } while (0)
-#define UOP_EXE_INC(opname) do { if (_Py_stats) _Py_stats->optimization_stats.opcode[opname].execution_count++; } while (0)
+#define UOP_STAT_INC(opname, name) do { if (_Py_stats) { assert(opname < 512); _Py_stats->optimization_stats.opcode[opname].name++; } } while (0)
```

Comment:
I would have named this arg `opcode` (or `OPCODE`, per recent Discourse thread), since to me `opname` is a variable of type `char *` giving the opcode's name; `opcode` is the (named) constant representing the opcode, which it is. (I did a small double take when I saw `assert(opname < 512)`. :-) But it's too late, and `opname` is used in several places above as well, so let's keep it this way.